templates/cassiopeia/index.php

```diff
@@ -28,6 +28,17 @@
 $menu     = $app->getMenu()->getActive();
 $pageclass = $menu->getParams()->get('pageclass_sfx');
 
+// Template path
+$templatePath = Uri::root() . 'templates/'.$this->template;
+
+// Color Theme
+$paramsColorName = $this->params->get('colorName');
+$assetColorName  = 'theme.' . $paramsColorName;
+$assetColorUri = $templatePath . '/css/global/' . $paramsColorName . '.css';
+//$wa->registerAndUseStyle('summer', 'templates/cassiopeia/css/global/summer.css');
+$wa->registerAndUseStyle($assetColorName, $assetColorUri);
```

Comment:
Please make sure, that the file exists (or does the WA does it automatically?)

Reply:
I think, it is done automatically, but I'm not sure